Dockerfile

```diff
@@ -36,7 +36,8 @@ CMD cd /mnt/ctlog/ && \
         --trusted_cert_file=/usr/local/etc/ctlog_ca_roots.pem \
         --stderrthreshold=0 \
         --log_dir=/mnt/ctlog/logs \
-        --tree_signing_frequency_seconds=60 \
+        --tree_signing_frequency_seconds=30 \
+        --guard_window_seconds=10 \
```

Comment:
Yeah, I'll pull it out into a separate PR though and merge it in anyway, then we can remove it once things have settled down a bit.